pinot-core/src/test/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPrunerTest.java

```diff
@@ -148,17 +158,18 @@ public void testBloomFilterInPredicatePruning() {
     when(indexSegment.getDataSource("column")).thenReturn(dataSource);
     // Add support for bloom filter
     DataSourceMetadata dataSourceMetadata = mock(DataSourceMetadata.class);
-    BloomFilterReader bloomFilterReader = mock(BloomFilterReader.class);
+    BloomFilterReader bloomFilterReader = new BloomFilterReaderBuilder()
```

Comment:
I have to remove the mock and use an actual object because the new implementation doesn't call `mightContain(String)` and instead calls `mightContain(long, long)`, which would make mocks quite more difficult to read.

This new implementation has the problem that false positives may affect the tests, but the bloom filter created is larger than required, so the possibility is very small. Even more, this false positives are deterministic. So even if one false positive is found in future, we can just change the examples to use a set of values that doesn't produce a collision.